packages/appstore/src/components/containers/listing-container.tsx

```diff
@@ -16,6 +16,37 @@ type ListingContainerProps = {
     className?: string;
     is_outside_grid_container?: boolean;
 };
+type OptionsProps = Pick<ListingContainerProps, 'title' | 'description' | 'is_deriv_platform'>;
+type SwitcherProps = Pick<ListingContainerProps, 'wallet_account' | 'is_deriv_platform'>;
+
+const Options = observer(({ title, description, is_deriv_platform }: OptionsProps) => {
+    const {
+        client: { is_landing_company_loaded },
+    } = useStore();
+
+    if (is_landing_company_loaded) {
```

Comment:
@sergei-deriv there is no difference between these to condition's return

Reply:
@mahdiyeh-deriv oh, really :) just copied it and didn't noticed. I'll change it now